src/payment/strategies/masterpass/masterpass-script-loader.ts

```diff
@@ -0,0 +1,22 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { StandardError } from '../../../common/error/errors';
+import { Masterpass, MasterpassHostWindow } from '../masterpass/masterpass';
+export default class MasterpassScriptLoader {
```

Comment:
Add an enter here.

Reply:
This is still not addressed.

Reply:
the 'enter' goes between ...scriptloader' and '{' ? or where?